config.py

```diff
@@ -0,0 +1,20 @@
+from enum import Enum
+
+class day(Enum):
+    sunday = 6
+    monday = 0
+    tuesday = 1
+    wednesday = 2
+    tursday = 3
+    friday = 4
+    saturday = 5
+
+class MessageType(Enum):
+    reminder = 0
+    NewPeriod = 1
+
+
+_Hour = 60*60
```

Comment:
these const values are used by other classes so I wouldn't underscore here.

Reply:
got it, dont know why i did that.